src/NewRelic.OpenTelemetry/NewRelicExporterHelperExtensions.cs

```diff
@@ -4,8 +4,8 @@
 using System;
 using System.Diagnostics;
 using Microsoft.Extensions.Logging;
+using NewRelic.OpenTelemetry;
 using NewRelic.Telemetry;
-using OpenTelemetry.Exporter.NewRelic;
 
 namespace OpenTelemetry.Trace
```

Comment:
Should we change this namespace too?

Reply:
The usage of the `OpenTelemetry.Trace` namespace is intentional here. The methods in this class are extension methods for `TracerProviderBuilder` which resides in the `OpenTelemetry.Trace` namespace. That way when someone has a reference to `NewRelic.OpenTelemetry` and adds a using for `OpenTelemetry.Trace` they're not left wondering why `AddNewRelicExporter` is not available.